### PR TITLE
2U Copy Edits for Weeks 1, 2, 3

### PR DESCRIPTION
**Pull request by 2hc1**
Punctuation, spelling, and grammatical copy-edit changes for Weeks 1, 2, 3

### CODE REVIEW

**Comment by 2hc1** [comment]
Hello Professor Guerra,

I've made copy-edit changes directly into the `index.pug` files for Weeks 01, 02, and 03. Your slides are great! Very excited to receive the next batch. Please let me know if you have any questions!

- Haley Cline

**Comment by 2hc1** [comment]
@john-guerra @preeves653 Ready for review and merge :)

**Comment by john-guerra** [comment]
Thanks a lot @2hc1 let me take a look

File: 03_Why/index.pug [hunk 1]
```diff
@@ -67,10 +67,10 @@ html
 
 
           section
-            h2 Why analyze tasks
+            h2 Why analyze tasks?
```
Comment (john-guerra) [inline 03_Why/index.pug:70]: This has to be: Why: Analyzing tasks

Comment (john-guerra) [reply 03_Why/index.pug:70]: I'm merging it and changing it on my side

**Comment by john-guerra** [comment]
These should be merged now and should be live in https://johnguerra.co/lectures/MIDS_W209_Information_Visualization/

**Pull request merged**